src/Configuration/PhotoboothConfiguration.php

```diff
@@ -997,13 +998,20 @@ protected function addAdminPanel(): NodeDefinition
 
     protected function addUiNode(): NodeDefinition
     {
+        include PathUtility::getAbsolutePath('/src/timezones.php');
```

Comment:
include PathUtility::getAbsolutePath('src/timezones.php');

Reply:
this timezone array is in this protected function not available